CaseStudy.md

```diff
@@ -0,0 +1,132 @@
+•	Each ML Model is packaged in an independent Docker Image
+•	Model verification before storing the Docker image
+•	All Docker images are stored in Azure Container Registry
+•	Builds and deploys Smoke Test module on Edge device
+
+### Use Case Builder Design 
+Use case builder provides a convention based approach to enable the developers to build and manage multiple use-cases across different environments. The framework provides the ability to create a single deployment manifest that bundles one or more ML models along the orchestration layer to server an end to end use case.
+
+At its core, the Use Case Builder leverages Azure IoT Edge and Azure DevOps pipelines to allow the developers build and integration test the orchestration layer in conjunction with the dependent ML Models on the targeted edge devices.
+
+The workflow of Use Case Builder from a developer environment to a production container registry is depicted below .
+  ![image](/docs/images/UseCaseBuilder_Design.png)
+#### Key Features
+•	Bundles model reference and orchestration layer into a single deployable entity based on the business scenario.
+•	DevOps pipeline templates for the use-cases to build, test and publish artifacts.
+•	Ability to author and manage the IoT Edge manifest for a use-case specific deployment on the targeted edge devices.
+•	Validation of the complete use-cases using an integrated end-to-end test running on the actual edge device.
+•	Provision to scaffold a use-case with a predefined folder structure.
+### Data Collection on the Edge Design
+Data Collection on the Edge component enables the customer to have a secure and automated process for data collection (images) from the edge devices, and use those images as a source for subsequent model training (post annotation).
+This component leverages [Azure Blob Storage IoT Edge](https://learn.microsoft.com/en-us/azure/iot-edge/how-to-store-data-blob?view=iotedge-1.4) modules at its core to provides a low-code mechanism for the edge devices to push binary data to remote azure blob storage accounts.
+ 
+ ![image](/docs/images/DataCollection_Design.png)
+
+#### Key Features
+- A no-code low-code solution which can be quickly setup to automate data collection on edge.
+- Supports intermittent internet connectivity and provides an asynchronous way for data upload from edge.
+- Support upload of high-quality large size images for model training on the cloud
+
+### Deployment Size Optimizations for Constrained Network
+To address key challenges around heavy edge images leading to exponential deployments (caused by multiple failures), lack of support for limited network bandwidth and high network cost, a two-stage approach was implemented.
+#### <ins>1.	Orchestrated Inferencing :</ins> The concept of Orchestrated inferencing was introduced which dictate the design based on decoupling the ML models and orchestration logic into separate docker containers. 
+
+ ![image](/docs/images/Orchestrated_Inferencing.png)
```

Comment:
nit: the width and height of this image can be reduced.

Reply:
Tried changing the width and the height of the image. Not sure if this is quite readable. Please review once.